content/livesync/connector/index.textile

```diff
@@ -17,7 +17,7 @@ docker pull ghcr.io/ably-labs/adbc:latest
 ```
 
 <aside data-type='note'>
-<p>Please "reach out":https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewformif you have any feedback or prefer to use an Ably-hosted version of LiveSync.</p>
+<p>Please "reach out":https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewform you have any feedback or prefer to use an Ably-hosted version of LiveSync.</p>
```

Comment:
I think the "if" should have been part of the sentence (in both places) 🙂

Reply:
Ah

Reply:
@m-hulbert I've addressed your feedback here: ecf2611848db5a62f6740d9d2173eb6357b6a274